events/iSawThat.ts

```diff
@@ -7,7 +7,18 @@ const ISawThat = {
   async execute(message: Message) {
     if (message.author.bot) return
     if (ignored.includes(message.author.id)) return
-    await message.channel.send(`i saw that <@${message.author.id}>`)
+    const logChannel = await message.guild.channels.cache.find(
+      channel => channel.name === 'logs'
+    ) as TextChannel
+
```

Comment:
put in check here if logChannel does not exist to just send i saw that and return
```js
if (!logChannel) {
  await message.channel.send(`i saw that <@${message.author.id}>`)
  return
}